### DISCUSSION TITLE
Support for Tidal?

### DISCUSSION THREAD

**Discussion by codefreak558** [Shabinder/SpotiFlyer#684] [Ideas]
Support to downloads Tidal songs.

You will have to log into your Tidal account to download higher qualities. Here is a working one that I use right now it would be cool to have it working here as well.

https://github.com/yaronzz/Tidal-Media-Downloader

**Comment by Shabinder** [comment]
will surely look into this

**Comment by sapreen555** [comment]
thank you

https://brwez.com/
